lib/galaxy/jobs/runners/util/job_script/DEFAULT_JOB_FILE_TEMPLATE.sh

```diff
@@ -4,6 +4,10 @@
 # to ensure this script is runnable before running it directly
 # or submitting it to a cluster manager.
 if [ -n "$ABC_TEST_JOB_SCRIPT_INTEGRITY_XYZ" ]; then
+    $command
```

Comment:
Command might not just be a single script, which I haven't checked but I imagine is why Travis isn't working.

If we are going to do this terrible hack again - I think we should just wait on the tool script itself - not have the job script depend on tool script logic. I can hack up a prototype of this if you wish.